packages/shared/lib/src/sdk/args.rs

```diff
@@ -335,21 +333,16 @@ pub fn vote_proposal_tx_args(
 
 #[derive(BorshSerialize, BorshDeserialize)]
 #[borsh(crate = "namada::core::borsh")]
-pub struct TransparentTransferMsg {
+pub struct TransferDataMsg {
```

Comment:
I'll take a look - I tried to remove `Transparent` where it could refer to another type of Transfer since it's more generic now, but when it comes to args you're right - the args will be different shielded. I think I got a little overzealous when renaming things :D

Reply:
I think for the file name in `types` I may keep everything in `types/src/tx/schema/transfer.ts` - we can just define multiple schemas there, including the first to be defined which should be `TransparentTransferMsgValue`

Reply:
Updated - and I think our `TxType` enum should still indicate `TransparentTransfer` as that determines what args are acceptable, so I'm changing that back as well!

Reply:
Done!